src/sdk/pynni/nni/compression/torch/utils/counter.py

```diff
@@ -111,22 +111,21 @@ def count_linear_mask(m, x, y):
     y : torch.Tensor
         output data
     """
-    output_channel = y.size()[1]
-    output_size = torch.zeros(y.size()[2:]).numel()
+    output_channel = y.numel()
```

Comment:
why the output channel is `y.numel()`?  should it be `m.out_features`?

Reply:
`m` is the `PrunerModuleWrapper`, `y` is the feature map after linear transformation, so the total number of elements calculated by `y.numel()` is the same of `m.out_features`.

Reply:
OK, got it.